lib/tty/box.rb

```diff
@@ -199,15 +208,7 @@ def frame(*content, top: nil, left: nil, width: nil, height: nil, align: :left,
       left_size   = border.left? ? 1 : 0
       right_size  = border.right ? 1 : 0
 
-      if block_given?
-        str = yield
-      else
-        str = case content.size
-        when 0 then ""
-        when 1 then content[0]
-        else content.join(NEWLINE)
-        end
-      end
+      str = block_given? ? yield : content_to_str(content)
```

Comment:
I like this change! Please see my other comment about moving the `content_to_str` method definition a bit closer.